universal/router/Root/index.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+import React from 'react'
+
+import Home from 'layouts/Home'
+
+const HomeRoute = () => (
+  <Home />
+)
```

Comment:
You're right we could...but i left this here so that any route specific logic like fetches, etc. could be incorporated here..we could omit this for now, but i also think it helps provide a decent middleground for not over-engineering while providing an initial starting point for customizing the route further. 

I honestly don't care...just trying to play both sides of the fence. You can make the call on this one :)